Sources/SitrepCore/FileVisitor.swift

```diff
@@ -203,8 +203,11 @@ class FileVisitor: SyntaxVisitor {
         let inheritanceClause = node.inheritanceClause?.inheritedTypeCollection.map {
             "\($0.typeName)".trimmingCharacters(in: .whitespacesAndNewlines)
         } ?? []
+        
+        let name = node.name
+            .trimmingCharacters(in: .whitespaces)
```

Comment:
this fixes an issue I've found where some names had an extra space at the end. I think this is because the way SwiftSyntax deals with leading and trailing tokens. 
Without this the `struct Foobar` name was `Foobar` but the `extension Foobar` name was `Foobar `. 
I could do this on the new `collate` code but I think here is more appropriate.